testing/src/main/java/org/oppia/android/testing/TextInputAction.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import android.view.View
+import androidx.annotation.StringRes
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import javax.inject.Inject
+
+class TextInputAction @Inject constructor(
+  val context: Context
+) {
+
```

Comment:
Add todo `// TODO(#1720): Move this to a companion object & use a test-only singleton injector to retrieve`

Also, Please check this #1720 and let me know if we can fix this file in this PR only if not, no worries.

Reply:
@anandwana001 
I didn't understand this issue. I guess I need to implement companion object of TextInputAction class

Reply:
PTAL

Reply:
I think we can skip this for now, this can be handle when we introduce dagger hilt.